python/ray/air/tests/test_api.py

```diff
@@ -141,6 +141,63 @@ def test_scaling_config_validate_config_bad_allowed_keys():
     assert "are not present in" in str(exc_info.value)
 
 
+def test_scaling_config_accelerator_type():
+    # Basic
+    scaling_config = ScalingConfig(num_workers=2, use_gpu=True, accelerator_type="A100")
+    assert scaling_config.accelerator_type == "A100"
+    assert scaling_config._trainer_resources_not_none == {
+        "CPU": 1,
+        "accelerator_type:A100": 0.001,
+    }
+    assert scaling_config._resources_per_worker_not_none == {
+        "GPU": 1,
+        "accelerator_type:A100": 0.001,
+    }
```

Comment:
Can we test this with the output of a public API instead? Maybe just getting the bundles of the `as_placement_group_factory` directly?